### PR TITLE
Upgrade guide 5.2.x - 5.3.0

### PR DESCRIPTION
**Pull request by htkassner**
Guess the "Other changes" part has to be filtered out.

### CODE REVIEW

File: source/developers-guide/shopware-5-upgrade-guide-for-developers/index.md [hunk 1]
```diff
@@ -17,6 +17,660 @@ including minor and bugfix releases, refer to the `UPGRADE.md` file found in you
...
+
+### CookiePermission
+
+Cookie permissions is now a part of shopware and you can configure it in the shop settings.
+
+We implement a basic cookie permission hint. If you want to change the decision whether the item is displayed or not, overwrite the jQuery plugin in the `jquery.cookie-permission.js`.
+
+### Shopping Worlds
+
+Shopping World have been technically refactored from the ground up to improve the overall performance when adding several elements to a shopping world. It is now possible to export and import shopping worlds via the backend.
```
Comment (jenskueper) [inline source/developers-guide/shopware-5-upgrade-guide-for-developers/index.md]: Worlds

File: source/developers-guide/shopware-5-upgrade-guide-for-developers/index.md [hunk 2]
```diff
@@ -17,6 +17,660 @@ including minor and bugfix releases, refer to the `UPGRADE.md` file found in you
...
+You can also convert shopping worlds to presets for reusability of configured shopping worlds. Please see Developer Docs article for further information.
```
Comment (jenskueper) [inline source/developers-guide/shopware-5-upgrade-guide-for-developers/index.md]: Vielleicht eher: Please look at the Developer Article for further information.

File: source/developers-guide/shopware-5-upgrade-guide-for-developers/index.md [hunk 3]
```diff
@@ -17,6 +17,660 @@ including minor and bugfix releases, refer to the `UPGRADE.md` file found in you
...
+**Before: Subscribe to an event and process element data in the callback method**
+
+```php
+public static function getSubscribedEvents()
+{
+    return ['Shopware_Controllers_Widgets_Emotion_AddElement' => 'handleSideviewElement'];
+}
+```
+
+**After: Create new class and tag it as `shopware_emotion.component_handler` in your `services.xml`**
```
Comment (jenskueper) [inline source/developers-guide/shopware-5-upgrade-guide-for-developers/index.md]: Create a new...

File: source/developers-guide/shopware-5-upgrade-guide-for-developers/index.md [hunk 4]
```diff
@@ -17,6 +17,660 @@ including minor and bugfix releases, refer to the `UPGRADE.md` file found in you
...
+    {
+        // do some handle logic and fill data
+        $element->getData()->set('key', 'value');
+    }
+}
+```
+
+#### Requesting items in ComponentHandler
+
+To make use of the performance improvement, you have to split your logic into a prepare step and handle step. The prepare step collects product numbers or criteria objects which will be resolved across all elements at once. The handle step provides a collection with resolved products and can be filled into your element.
```
Comment (jenskueper) [inline source/developers-guide/shopware-5-upgrade-guide-for-developers/index.md]: vielleicht lieber merged anstatt filled?

File: source/developers-guide/shopware-5-upgrade-guide-for-developers/index.md [hunk 5]
```diff
@@ -17,6 +17,660 @@ including minor and bugfix releases, refer to the `UPGRADE.md` file found in you
...
+* Added class `.filter-panel--checkbox` to `themes/Frontend/Responsive/frontend/_public/src/less/_components/filter-panel.less`
+* Added class `.radio--state` to `themes/Frontend/Responsive/frontend/_public/src/less/_components/filter-panel.less`
+* Added class `.checkbox--state` to `themes/Frontend/Responsive/frontend/_public/src/less/_components/filter-panel.less`
+* Added JavaScript method `document.asyncReady()` to register callbacks which fire after the main script was loaded asynchronously.
+* Added missing dependency `jquery.event.move` to the `package.json` file.
+* Added template switch for `listing/index.tpl` to `listing/customer_stream.tpl` in case that the category contains a shopping world which is restricted to customer streams
+* Added database column `s_emarketing_vouchers.customer_stream_ids` to restrict vouchers to customer streams.
+* Added database column `s_emotion.customer_stream_ids` to restrict shopping worlds to customer streams.
+* Added database table `s_customer_streams` for a list of all existing streams (`Shopware\Models\Customer\CustomerStream`)
+* Added database table `s_customer_search_index` for an fast customer search
```
Comment (jenskueper) [inline source/developers-guide/shopware-5-upgrade-guide-for-developers/index.md]: a

File: source/developers-guide/shopware-5-upgrade-guide-for-developers/index.md [hunk 6]
```diff
@@ -17,6 +17,660 @@ including minor and bugfix releases, refer to the `UPGRADE.md` file found in you
...
+    * `Shopware\Components\Emotion\Preset\ComponentHandler\Html5VideoComponentHandler`
+* Added new ExtJs views for emotion presets under `themes\backend\emotion\view\preset`
+* Added new service tag for registering emotion preset component handlers `shopware.emotion.preset_component_handler`
+* Added actions to import and export shopping worlds in `Shopware_Controllers_Backend_Emotion`
+* Changed theme path for new plugins from `/resources` into `/Resources`
+* Changed sorting of `Shopware.listing.FilterPanel` fields
+* Changed database column `s_articles_vote`.`answer_date` to allow `NULL` values
+* Changed `LastArticle` plugin config elements `show`, `controller` and `time` to be prefixed with `lastarticles_`
+* Changed product listings in shopping worlds to only be loaded if `showListing` is true
+* Changed sql query in `sAdmin` queries which uses a sub query for address compatibility, following functions affected:
```
Comment (jenskueper) [inline source/developers-guide/shopware-5-upgrade-guide-for-developers/index.md]: Der Satz ergibt keinen richtigen Sinn.

File: source/developers-guide/shopware-5-upgrade-guide-for-developers/index.md [hunk 7]
```diff
@@ -17,6 +17,660 @@ including minor and bugfix releases, refer to the `UPGRADE.md` file found in you
...
+* Smarty modifier `rewrite`
+* Vendor prefixes `-ms` and  `-o` from all mixins in the Bare and Responsive Theme
+* Scrollbar styling on filter-panels (Selector: `.filter-panel--content`)
+* Support for `.swf` file type in the banner module
+* `max-width` rule for `.filter--active` in `themes/Frontend/Responsive/frontend/_public/src/less/_components/filter-panel.less`
+   
+### Other changes
+* Added config element `displayOnlySubShopVotes` to display only shop assigned article votes
+* Added parameter `displayProgressOnSingleDelete` to `Shopware.grid.Panel` to hide progress window on single delete action
+* Added parameter `expression` in `Shopware.listing.FilterPanel` to allow define own query expressions
```
Comment (jenskueper) [inline source/developers-guide/shopware-5-upgrade-guide-for-developers/index.md]: to allow the definition of own query expressions

File: source/developers-guide/shopware-5-upgrade-guide-for-developers/index.md (hunk 6 above)
Comment (htkassner) [inline source/developers-guide/shopware-5-upgrade-guide-for-developers/index.md]: @jenskueper These are snippets from the upgrade.md. But this is a good catch. We have to evaluate which are necessary and which maybe can be left out.

File: source/developers-guide/shopware-5-upgrade-guide-for-developers/index.md [hunk 8]
```diff
@@ -17,6 +17,673 @@ including minor and bugfix releases, refer to the `UPGRADE.md` file found in you
...
+    <select>
+        <option></option>
+        <option></option>
+    </select>
+</div>
+```
+
+### Batch Product Search
+
+The Batch Product Search service works with request and results. You can add multiple criteria's and/or product numbers to a request and resolve them in an optimized way. An optimizer groups multiple equal criteria's into one and performs the search.
```
Comment (soebbing) [inline source/developers-guide/shopware-5-upgrade-guide-for-developers/index.md]: "criteria's" should spell "criteria" - that already is the plural form.

File: source/developers-guide/shopware-5-upgrade-guide-for-developers/index.md [hunk 9]
```diff
@@ -17,6 +17,673 @@ including minor and bugfix releases, refer to the `UPGRADE.md` file found in you
...
+    Criteria $reverted,
+    Criteria $criteria,
+    ShopContextInterface $context
+) {
+    $query = $this->queryBuilderFactory->createQuery($reverted, $context);
+    //...
+```
+
+#### Elastic search
+In the elastic search implementation the current filter behavior is controlled by the condition handlers. By adding an query as `post filter`, facets are not affected by this filter.
```
Comment (soebbing) [inline source/developers-guide/shopware-5-upgrade-guide-for-developers/index.md]: "...by adding *a* query..."

File: source/developers-guide/shopware-5-upgrade-guide-for-developers/index.md [hunk 10]
```diff
@@ -17,6 +17,673 @@ including minor and bugfix releases, refer to the `UPGRADE.md` file found in you
...
+    * `frontend_listing_actions_sort_field_relevance`
+    * `frontend_listing_actions_sort_field_release`
+    * `frontend_listing_actions_sort_field_rating`
+    * `frontend_listing_actions_sort_field_price_asc`
+    * `frontend_listing_actions_sort_field_price_desc`
+    * `frontend_listing_actions_sort_field_name`
+    * `frontend_search_category_filter`
+    * `frontend_listing_swf_banner`
+
+#### jQuery plugins
```
Comment (Phil23) [inline source/developers-guide/shopware-5-upgrade-guide-for-developers/index.md]: Hier wird noch einiges fehlen. Zum Beispiel die Filter im Listing.

File: source/developers-guide/shopware-5-upgrade-guide-for-developers/index.md [hunk 11]
```diff
@@ -17,6 +17,673 @@ including minor and bugfix releases, refer to the `UPGRADE.md` file found in you
...
+}
+```
+
+Keep in mind to use a unique key for requesting and getting products. For best practise, use the element's id in your key (`$element->getId()`).
+
+### Library updates
+
+* Updated `FPDF` to 1.8.1
+* Updated `FPDI` to 1.6.1
+* Updated `flatpickr` to 2.5.7
```
Comment (philipgatzka) [inline source/developers-guide/shopware-5-upgrade-guide-for-developers/index.md]: `flatpickr` is still version 2.4.7

File: source/developers-guide/shopware-5-upgrade-guide-for-developers/index.md [hunk 12]
```diff
@@ -17,6 +17,673 @@ including minor and bugfix releases, refer to the `UPGRADE.md` file found in you
...
+    </select>
+</div>
+```
+
+### Batch Product Search
+
+The Batch Product Search service works with request and results. You can add multiple criteria's and/or product numbers to a request and resolve them in an optimized way. An optimizer groups multiple equal criteria's into one and performs the search.
+
+```php
+$criteria = new Critera();
```
Comment (philipgatzka) [inline source/developers-guide/shopware-5-upgrade-guide-for-developers/index.md]: `new Criter`i`a();`

File: source/developers-guide/shopware-5-upgrade-guide-for-developers/index.md [hunk 13]
```diff
@@ -17,6 +17,673 @@ including minor and bugfix releases, refer to the `UPGRADE.md` file found in you
...
+$criteria->addCondition(new CategoryCondition([3]));
+$criteria->limit(3);
+
+$anotherCriteria = new Critera();
```
Comment (philipgatzka) [inline source/developers-guide/shopware-5-upgrade-guide-for-developers/index.md]: `new Criter`i`a();`

File: source/developers-guide/shopware-5-upgrade-guide-for-developers/index.md [hunk 14]
```diff
@@ -17,6 +17,673 @@ including minor and bugfix releases, refer to the `UPGRADE.md` file found in you
...
+* Changed sorting of `Shopware.listing.FilterPanel` fields
+* Changed database column `s_articles_vote`.`answer_date` to allow `NULL` values
+* Changed `LastArticle` plugin config elements `show`, `controller` and `time` to be prefixed with `lastarticles_`
+* Changed product listings in shopping worlds to only be loaded if `showListing` is true
+* Changed sql query
+ in `sAdmin` queries which uses a sub query for address compatibility, following functions affected:
+    * `sAdmin::sGetDispatchBasket`
+    * `sAdmin::sGetPremiumDispatches`
+    * `sAdmin::sGetPremiumDispatchSurcharge`
+* Changed attribute type `string` mapping to mysql `TEXT` type. String and single selection data type supports no longer a sql default value.
```
Comment (philipgatzka) [inline source/developers-guide/shopware-5-upgrade-guide-for-developers/index.md]: "...supports no longer..." -> "...does no longer support..."

File: source/developers-guide/shopware-5-upgrade-guide-for-developers/index.md (hunk 5 above)
Comment (htkassner) [inline source/developers-guide/shopware-5-upgrade-guide-for-developers/index.md]: Fixed

File: source/developers-guide/shopware-5-upgrade-guide-for-developers/index.md (hunk 8 above)
Comment (htkassner) [inline source/developers-guide/shopware-5-upgrade-guide-for-developers/index.md]: Fixed

File: source/developers-guide/shopware-5-upgrade-guide-for-developers/index.md (hunk 9 above)
Comment (htkassner) [inline source/developers-guide/shopware-5-upgrade-guide-for-developers/index.md]: Fixed

File: source/developers-guide/shopware-5-upgrade-guide-for-developers/index.md (hunk 11 above)
Comment (htkassner) [inline source/developers-guide/shopware-5-upgrade-guide-for-developers/index.md]: Fixed

File: source/developers-guide/shopware-5-upgrade-guide-for-developers/index.md (hunk 13 above)
Comment (htkassner) [inline source/developers-guide/shopware-5-upgrade-guide-for-developers/index.md]: Fixed

File: source/developers-guide/shopware-5-upgrade-guide-for-developers/index.md (hunk 12 above)
Comment (htkassner) [inline source/developers-guide/shopware-5-upgrade-guide-for-developers/index.md]: Fixed

File: source/developers-guide/shopware-5-upgrade-guide-for-developers/index.md [hunk 15]
```diff
@@ -17,6 +17,673 @@ including minor and bugfix releases, refer to the `UPGRADE.md` file found in you
...
+    Criteria $criteria,
+    ShopContextInterface $context
+) {
+    $query = $this->queryBuilderFactory->createQuery($reverted, $context);
+    //...
+```
+
+#### Elastic search
+In the elastic search implementation the current filter behavior is controlled by the condition handlers. By adding an query as `post filter`, facets are not affected by this filter.
+This behavior is checked over the `Criteria->hasBaseCondition` statement:
```
Comment (htkassner) [inline source/developers-guide/shopware-5-upgrade-guide-for-developers/index.md]: Fixed

File: source/developers-guide/shopware-5-upgrade-guide-for-developers/index.md [hunk 16]
```diff
@@ -17,6 +17,673 @@ including minor and bugfix releases, refer to the `UPGRADE.md` file found in you
...
+        } else {
+            $this->handlePostFilter($criteriaPart, $criteria, $search, $context);
+        }
+    }
+}
+```
+
+### CookiePermission
+
+Cookie permissions is now a part of shopware and you can configure it in the shop settings.
```
Comment (htkassner) [inline source/developers-guide/shopware-5-upgrade-guide-for-developers/index.md]: Fixed

File: source/developers-guide/shopware-5-upgrade-guide-for-developers/index.md [hunk 17]
```diff
@@ -17,6 +17,673 @@ including minor and bugfix releases, refer to the `UPGRADE.md` file found in you
...
+
+We implement a basic cookie permission hint. If you want to change the decision whether the item is displayed or not, overwrite the jQuery plugin in the `jquery.cookie-permission.js`.
+
+### Shopping worlds
+
+Shopping worlds have been technically refactored from the ground up to improve the overall performance when adding several elements to a shopping world. It is now possible to export and import shopping worlds via the backend.
+You can also convert shopping worlds to presets for reusability of configured shopping worlds. Please look at the Developer Docs article for further information.
```
Comment (htkassner) [inline source/developers-guide/shopware-5-upgrade-guide-for-developers/index.md]: Doesn't exist yet. Will be merged soon.

File: source/developers-guide/shopware-5-upgrade-guide-for-developers/index.md [hunk 18]
```diff
@@ -17,6 +17,673 @@ including minor and bugfix releases, refer to the `UPGRADE.md` file found in you
...
+
+#### ComponentHandler
+
+The processing of elements has been changed from events to classes of component handler.
```
Comment (htkassner) [inline source/developers-guide/shopware-5-upgrade-guide-for-developers/index.md]: Fixed

File: source/developers-guide/shopware-5-upgrade-guide-for-developers/index.md [hunk 19]
```diff
@@ -17,6 +17,673 @@ including minor and bugfix releases, refer to the `UPGRADE.md` file found in you
 
 <div class="toc-list"></div>
 
+## Shopware 5.3
+
+### System requirements changes
+
+The minimum PHP version is still **PHP 5.6.4 or higher**.
```
Comment (soebbing) [inline source/developers-guide/shopware-5-upgrade-guide-for-developers/index.md]: "...still is..."

File: source/developers-guide/shopware-5-upgrade-guide-for-developers/index.md [hunk 20]
```diff
@@ -17,6 +17,673 @@ including minor and bugfix releases, refer to the `UPGRADE.md` file found in you
...
+
+### Smarty Rendering
+
+#### Form Fields
```
Comment (soebbing) [inline source/developers-guide/shopware-5-upgrade-guide-for-developers/index.md]: Form Module in the Backend

File: source/developers-guide/shopware-5-upgrade-guide-for-developers/index.md [hunk 21]
```diff
@@ -17,6 +17,673 @@ including minor and bugfix releases, refer to the `UPGRADE.md` file found in you
...
+
+Only variables that were previously assigned to the view are rendered. In addition, smarty function calls are no longer executed.
```
Comment (soebbing) [inline source/developers-guide/shopware-5-upgrade-guide-for-developers/index.md]: Smarty functions in form templates have been disabled.  Also no new variables can be added to the template.

File: source/developers-guide/shopware-5-upgrade-guide-for-developers/index.md [hunk 22]
```diff
@@ -17,6 +17,673 @@ including minor and bugfix releases, refer to the `UPGRADE.md` file found in you
...
+{sElement.name} // works
+
+{sElement.name|currency} // works, but does not execute the currency function
+
+{sElement.value[$key]|currency} // does not work
+```
+
+#### Tracking Code
+
+Smarty rendering has been disable for this section. All variables have been removed with one exception. The variable `{$offerPosition.trackingcode}` is a placeholder now. To generate tracking urls, use the following pattern:
```
Comment (soebbing) [inline source/developers-guide/shopware-5-upgrade-guide-for-developers/index.md]: "...has been disabled..."

File: source/developers-guide/shopware-5-upgrade-guide-for-developers/index.md [hunk 23]
```diff
@@ -17,6 +17,673 @@ including minor and bugfix releases, refer to the `UPGRADE.md` file found in you
...
+| Field | Snippet name |
+|-------|--------------|
+| Snippet namespace         |  backend/attribute_columns |
+| Snippet name label        |  s_articles_attributes_attr1_label |
+| Snippet name support text |  s_articles_attributes_attr1_supportText |
+| Snippet name help text    |  s_articles_attributes_attr1_helpText |
+
+### Backend Components
+
+You can now define the expression for the comparison in SQL. For example `>=` like seen below:
```
Comment (soebbing) [inline source/developers-guide/shopware-5-upgrade-guide-for-developers/index.md]: "For example `>=` can be defined like seen below:"

File: source/developers-guide/shopware-5-upgrade-guide-for-developers/index.md [hunk 24]
```diff
@@ -17,6 +17,673 @@ including minor and bugfix releases, refer to the `UPGRADE.md` file found in you
...
+$result = $this->container->get('shopware_search.batch_product_search')->search($request, $context);
+
+$result->get('numbers-1'); // ['SW10004' => ListProduct, 'SW10006' => ListProduct]
+$result->get('criteria-1'); // ['SW10006' => ListProduct, 'SW10007' => ListProduct, 'SW10008' => ListProduct]
+$result->get('criteria-2'); // ['SW10009' => ListProduct, 'SW10010' => ListProduct, 'SW10011' => ListProduct, 'SW10012' => ListProduct, 'SW10013' => ListProduct]
+```
+
+### Partial facets
+
+`\Shopware\Bundle\SearchBundleDBAL\FacetHandlerInterface` marked as deprecated and replaced by `\Shopware\Bundle\SearchBundleDBAL\PartialFacetHandlerInterface`.
```
Comment (soebbing) [inline source/developers-guide/shopware-5-upgrade-guide-for-developers/index.md]: "..._is_ marked as ..."

File: source/developers-guide/shopware-5-upgrade-guide-for-developers/index.md [hunk 25]
```diff
@@ -17,6 +17,673 @@ including minor and bugfix releases, refer to the `UPGRADE.md` file found in you
...
+Each facet handler had to revert the provided criteria by their own to remove customer conditions. This behaviour is now handled in the `\Shopware\Bundle\SearchBundleDBAL\ProductNumberSearch::createFacets`
```
Comment (soebbing) [inline source/developers-guide/shopware-5-upgrade-guide-for-developers/index.md]: "..._on_ their own..."

File: source/developers-guide/shopware-5-upgrade-guide-for-developers/index.md [hunk 26]
```diff
@@ -17,6 +17,673 @@ including minor and bugfix releases, refer to the `UPGRADE.md` file found in you
...
+
+<div class="alert alert-info" role="alert">
+    <strong>Note:</strong> This section covers only the most relevant removals. Please refer to the UPGRADE.MD file in your Shopware installation for a complete, detailed list of removed elements
+</div>
+
+#### Article related
+* Default plugin `LastArticle`, use `shopware.components.last_articles_subscriber` instead
+* Session key `sLastArticle`
+* View variable `sLastActiveArticle` from basket
+* Join to `s_core_tax` in `Shopware\Bundle\SearchBundleDBAL\ProductNumberSearch`
```
Comment (soebbing) [inline source/developers-guide/shopware-5-upgrade-guide-for-developers/index.md]: "Join _on_..."

File: source/developers-guide/shopware-5-upgrade-guide-for-developers/index.md [hunk 27]
```diff
@@ -17,6 +17,673 @@ including minor and bugfix releases, refer to the `UPGRADE.md` file found in you
...
+    
+#### View variables
+* Global
+    * `hasEscapedFragment`
+* Forms
+    * `{$sShopname}`, use `{sShopname}` instead    
+    
+#### Frontend
+* Support for `Internet Explorer < 11`
+* Unneeded `css` classes.
```
Comment (soebbing) [inline source/developers-guide/shopware-5-upgrade-guide-for-developers/index.md]: The dot in the end should be removed

File: source/developers-guide/shopware-5-upgrade-guide-for-developers/index.md [hunk 28]
```diff
@@ -17,6 +17,673 @@ including minor and bugfix releases, refer to the `UPGRADE.md` file found in you
...
+* Changed template file `themes/Frontend/Bare/frontend/detail/index.tpl` to split it into separated files
+* Changed property `linkDetails` of `$sArticle`
+* Changed the article url to also contain the order number of the product
+* Changed the product selection to variant selection in `Shopware.apps.Emotion.view.components.BannerMapping`
+* Changed the integration of `modernizr.js` and added it to the compressed main JavaScript files
+* Changed the script tag for the generated JavaScript file for asynchronous loading, can be changed in theme configuration
+* Changed the inline script for the statistics update to vanilla JavaScript
+* Changed event name from `plugin/swAjaxProductNavigation/onSetProductState` to `plugin/swAjaxProductNavigation/onGetProductState`
+* Change behavior of the smarty rendering in forms fields comment. See below for more information
+* Change behavior of the tracking url rendering. See below for more information
```
Comment (soebbing) [inline source/developers-guide/shopware-5-upgrade-guide-for-developers/index.md]: Change "Change" to "Changed" .  And I don't see where the more information are.

File: source/developers-guide/shopware-5-upgrade-guide-for-developers/index.md [hunk 29]
```diff
@@ -17,6 +17,673 @@ including minor and bugfix releases, refer to the `UPGRADE.md` file found in you
...
+* Changed smarty blocks from `frontend_forms_elements*` to `frontend_forms_form_elements*`
+* Changed template file `themes/Frontend/Bare/frontend/detail/index.tpl` to split it into separated files
+* Changed property `linkDetails` of `$sArticle`
+* Changed the article url to also contain the order number of the product
+* Changed the product selection to variant selection in `Shopware.apps.Emotion.view.components.BannerMapping`
+* Changed the integration of `modernizr.js` and added it to the compressed main JavaScript files
+* Changed the script tag for the generated JavaScript file for asynchronous loading, can be changed in theme configuration
+* Changed the inline script for the statistics update to vanilla JavaScript
+* Changed event name from `plugin/swAjaxProductNavigation/onSetProductState` to `plugin/swAjaxProductNavigation/onGetProductState`
+* Change behavior of the smarty rendering in forms fields comment. See below for more information
```
Comment (soebbing) [inline source/developers-guide/shopware-5-upgrade-guide-for-developers/index.md]: Change "Change" to "Changed" .

File: source/developers-guide/shopware-5-upgrade-guide-for-developers/index.md (hunk 10 above)
Comment (htkassner) [inline source/developers-guide/shopware-5-upgrade-guide-for-developers/index.md]: Wie besprochen

File: source/developers-guide/shopware-5-upgrade-guide-for-developers/index.md (hunk 19 above)
Comment (htkassner) [inline source/developers-guide/shopware-5-upgrade-guide-for-developers/index.md]: Fixed

File: source/developers-guide/shopware-5-upgrade-guide-for-developers/index.md (hunk 20 above)
Comment (htkassner) [inline source/developers-guide/shopware-5-upgrade-guide-for-developers/index.md]: Fixed

File: source/developers-guide/shopware-5-upgrade-guide-for-developers/index.md (hunk 21 above)
Comment (htkassner) [inline source/developers-guide/shopware-5-upgrade-guide-for-developers/index.md]: Fixed

File: source/developers-guide/shopware-5-upgrade-guide-for-developers/index.md (hunk 22 above)
Comment (htkassner) [inline source/developers-guide/shopware-5-upgrade-guide-for-developers/index.md]: Fixed

File: source/developers-guide/shopware-5-upgrade-guide-for-developers/index.md (hunk 23 above)
Comment (htkassner) [inline source/developers-guide/shopware-5-upgrade-guide-for-developers/index.md]: Fixed

File: source/developers-guide/shopware-5-upgrade-guide-for-developers/index.md (hunk 24 above)
Comment (htkassner) [inline source/developers-guide/shopware-5-upgrade-guide-for-developers/index.md]: Fixed

File: source/developers-guide/shopware-5-upgrade-guide-for-developers/index.md (hunk 25 above)
Comment (htkassner) [inline source/developers-guide/shopware-5-upgrade-guide-for-developers/index.md]: Fixed

File: source/developers-guide/shopware-5-upgrade-guide-for-developers/index.md (hunk 26 above)
Comment (htkassner) [inline source/developers-guide/shopware-5-upgrade-guide-for-developers/index.md]: Fixed

File: source/developers-guide/shopware-5-upgrade-guide-for-developers/index.md (hunk 27 above)
Comment (htkassner) [inline source/developers-guide/shopware-5-upgrade-guide-for-developers/index.md]: Fixed

File: source/developers-guide/shopware-5-upgrade-guide-for-developers/index.md (hunk 29 above)
Comment (htkassner) [inline source/developers-guide/shopware-5-upgrade-guide-for-developers/index.md]: Number of the beast fixed

File: source/developers-guide/shopware-5-upgrade-guide-for-developers/index.md (hunk 28 above)
Comment (htkassner) [inline source/developers-guide/shopware-5-upgrade-guide-for-developers/index.md]: Fixed

**Pull request merged**